core-bundle/contao/templates/_new/content_element/download.html.twig

```diff
@@ -1,12 +1,10 @@
 {% extends "@Contao/content_element/_base.html.twig" %}
 {% use "@Contao/component/_download.html.twig" %}
 
-{% if downloads %}
-    {% set attributes = attrs(attributes|default).addClass(['download-element', "ext-#{(downloads|first).file.extension(true)}"]) %}
-{% endif %}
-
 {% block content %}
     {% if downloads %}
-        {% with {download: downloads|first} %}{{ block('download_component') }}{% endwith %}
+        <p{{ attrs(download_attributes|default).addClass(['download-element', "ext-#{(downloads|first).file.extension(true)}"]) }}>
```

Comment:
Not sure if @m-vo removed the `<p>` wrapper deliberately?

Reply:
Ah, there is actually an issue. The `_download` component also outputs the `<figure>` for the preview image, thus we cannot wrap it in `<p>`.